.github/workflows/deploy_module.yml

```diff
@@ -20,11 +20,11 @@ jobs:
       matrix:
         include:
           - arch: buildjet-8vcpu-ubuntu-2204
-            image: ghcr.io/viamrobotics/canon:amd64-cache
-            platform: amd64
+            image: ghcr.io/viamrobotics/rdk-devenv:amd64-cache
+            platform: linux/amd64
```

Comment:
this may fail when you run it -- this is being consumed elsewhere in the file as `matrix.platform`, both here + in the `upload` job

Reply:
Made consistent with other workflows